core/src/main/scala/kafka/admin/LogDirsCommand.scala

```diff
@@ -39,19 +39,29 @@ object LogDirsCommand {
     def describe(args: Array[String], out: PrintStream): Unit = {
         val opts = new LogDirsCommandOptions(args)
         val adminClient = createAdminClient(opts)
-        val topicList = opts.options.valueOf(opts.topicListOpt).split(",").filter(!_.isEmpty)
-        val brokerList = Option(opts.options.valueOf(opts.brokerListOpt)) match {
-            case Some(brokerListStr) => brokerListStr.split(',').filter(!_.isEmpty).map(_.toInt)
-            case None => adminClient.describeCluster().nodes().get().asScala.map(_.id()).toArray
-        }
+        val topicList = opts.options.valueOf(opts.topicListOpt).split(",").filter(_.nonEmpty)
```

Comment:
the resource declaration should be followed by `try` block.
```scala
val adminClient = createAdminClient(opts)
try {

}
```

Reply:
Good.It has been modified in the latest submission.